### PR TITLE
Update the "Enabling multidex support" section

### PR DESCRIPTION
**Pull request by anderdobo**
_Description of what this PR is changing or adding, and why:_ Updates the "Enabling multidex support" section to show _how_ to use the Flutter tool to enable Android multidex support.

_Issues fixed by this PR (if any):_ https://github.com/flutter/website/issues/8527

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — COMMENTED** [review]
I'm guessing the images referenced from this PR are in https://github.com/flutter/website/pull/8528?

Please merge the two PRs.

**Comment by domesticmouse** [comment]
It looks like the image assets are in the wrong place, according to the [linkcheck CI](https://github.com/flutter/website/actions/runs/4653827046/jobs/8235032765?pr=8529#step:3:1064).

PTAL @parlough @khanhnwin

File: src/deployment/android.md
Comment (domesticmouse) [inline src/deployment/android.md:263]: Should this be `<img src='/assets/images/docs/android/ide-build-failure-multidex.png'` instead?

Alternatively, move the image assets to match the above code?

Comment (anderdobo) [reply src/deployment/android.md:263]: I'll move the images to keep the paths of the links in the file consistent.

**Review by domesticmouse — APPROVED** [review]
LGTM

File: src/deployment/android.md
Comment (sfshaza2) [inline src/deployment/android.md:263]: It won't fail if you run it from the command line?

```suggestion
When you try to build and run your app with the **Run** and **Debug**
options in your IDE, your build might fail with the following message:
<img src='/assets/images/docs/deployment/android/ide-build-failure-multidex.png' width="100%" alt='screenshot of build failure because Multidex support is required'>
```

File: src/deployment/android.md
Comment (sfshaza2) [inline src/deployment/android.md:270]:
```suggestion
To enable multidex from the command line, run `flutter run --debug` and select an Android device:

<img src='/assets/images/docs/deployment/android/cli-select-device.png' width="100%" alt='screenshot of selecting an Android device'>

When prompted, enter `y`. The Flutter tool enables multidex support and retries the build:

<img src='/assets/images/docs/deployment/android/cli-multidex-added-build.png' width="100%" alt='screenshot of a successful build after adding multidex'>

```

File: src/deployment/android.md
Comment (sfshaza2) [inline src/deployment/android.md:273]:
```suggestion
  However, we don't recommend targeting API 21+ purely to resolve the
  multidex issue.
```

**Review by sfshaza2 — APPROVED** [review]
lgtm once a few tweaks are merged. I looked at the current live page and it looks like a tech writer never reviewed this!

**Pull request merged**
